### PR TITLE
Cross validation how to

### PR DESCRIPTION
**Pull request by natke**
Cross validation how-to

Re-open #11792, with changes
[Internal Review URL](https://review.docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/train-machine-learning-model-cross-validation-ml-net?branch=pr-en-us-12159)
[AB#1463694](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1463694)

Addresses:

#10956

### CODE REVIEW

File: docs/machine-learning/how-to-guides/train-machine-learning-model-cross-validation-ml-net.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    [VectorType(3)]
+    public float[] HistoricalPrices { get; set; }
+
+    [LoadColumn(4)]
+    [ColumnName("Label")]
+    public float CurrentPrice { get; set; }
+}
+```
+
+Load the data in into an [`IDataView`](xref:Microsoft.ML.IDataView).
```
Comment (JRAlexander) [inline docs/machine-learning/how-to-guides/train-machine-learning-model-cross-validation-ml-net.md]: Is code missing for this?

**Review by JRAlexander — CHANGES_REQUESTED** [review]
One clarfication, otherwise LGTM.

File: docs/machine-learning/how-to-guides/train-machine-learning-model-cross-validation-ml-net.md (hunk 1 above)
Comment (luisquintanilla) [reply docs/machine-learning/how-to-guides/train-machine-learning-model-cross-validation-ml-net.md]: No. Just an instruction to do so.

File: docs/machine-learning/how-to-guides/train-machine-learning-model-cross-validation-ml-net.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+title: "How-To: Learn how to train and evaluate a machine learning model using cross validation in ML.NET"
+description: Learn how to train and evaluate a machine learning model using cross validation in ML.NET
+ms.date: 05/01/2019
+author: luisquintanilla
+ms.author: luquinta
+ms.custom: mvc,how-to
+#Customer intent: As a developer, I want to use cross validation to train a machine learning model in ML.NET
+---
+
+# How-To: Learn how to train and evaluate a machine learning model using cross validation in ML.NET
```
Comment (luisquintanilla) [inline docs/machine-learning/how-to-guides/train-machine-learning-model-cross-validation-ml-net.md]:
```suggestion
Learn how to train and evaluate a machine learning model using cross validation in ML.NET
```

File: docs/machine-learning/how-to-guides/train-machine-learning-model-cross-validation-ml-net.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
+---
+title: "How-To: Learn how to train and evaluate a machine learning model using cross validation in ML.NET"
```
Comment (luisquintanilla) [inline docs/machine-learning/how-to-guides/train-machine-learning-model-cross-validation-ml-net.md]:
```suggestion
title: Learn how to train and evaluate a machine learning model using cross validation in ML.NET
```

**Review by luisquintanilla — APPROVED** [review]
Looks good. Minor suggestions

**Review by JRAlexander — APPROVED** [review]

File: docs/machine-learning/how-to-guides/train-machine-learning-model-cross-validation-ml-net.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+title: Learn how to train and evaluate a machine learning model using cross validation in ML.NET
+description: Learn how to train and evaluate a machine learning model using cross validation in ML.NET
+ms.date: 05/01/2019
+author: luisquintanilla
+ms.author: luquinta
+ms.custom: mvc,how-to
+#Customer intent: As a developer, I want to use cross validation to train a machine learning model in ML.NET
+---
+
+Learn how to train and evaluate a machine learning model using cross validation in ML.NET
```
Comment (luisquintanilla) [inline docs/machine-learning/how-to-guides/train-machine-learning-model-cross-validation-ml-net.md]:
```suggestion
# Learn how to train and evaluate a machine learning model using cross validation in ML.NET
```

**Review by luisquintanilla — APPROVED** [review]

**Pull request merged**